app/simplifiedAccountRoutes.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const { Router } = require('express')
+
+const getSimplifiedAccount = require('./middleware/simplified-account/simplified-account-strategy.middleware')
+const isOptedInToSimplifiedAccounts = require('./middleware/simplified-account/simplified-account-opt-in.middleware')
+const userIsAuthorised = require('./middleware/user-is-authorised')
+const paths = require('./paths')
+const serviceSettingsController = require('./controllers/simplified-account/settings')
+const permission = require('./middleware/permission')
+
+const simplifiedAccount = new Router({ mergeParams: true })
```

Comment:
could just make this the default export and remove the `exports` declaration at the end of the file

Reply:
hmm not sure how to make this work, can you give a code example?